spec/lib/git_reflow/config_spec.rb

```diff
@@ -6,8 +6,24 @@
     it      { expect{ subject }.to have_run_command_silently 'git config --get chucknorris.roundhouse' }
 
     context "and getting all values" do
-      subject { GitReflow::Config.get('chucknorris.roundhouse', all: true) }
-      it      { expect{ subject }.to have_run_command_silently 'git config --get-all chucknorris.roundhouse' }
+      subject { GitReflow::Config.get('chucknorris.roundhouse-kick', all: true) }
+      it      { expect{ subject }.to have_run_command_silently 'git config --get-all chucknorris.roundhouse-kick' }
+
+      context "and checking locally only" do
```

Comment:
When chucknorris jumps, does he jump off the planet, or does the planet move away from chucknorris?